test/msg.go

```diff
@@ -84,10 +84,6 @@ func (m *MockMsg) WithURNAuthTokens(tokens map[string]string) courier.Msg {
 }
 func (m *MockMsg) WithReceivedOn(date time.Time) courier.Msg { m.receivedOn = &date; return m }
 
-// used for testing created incoming messages
-func (m *MockMsg) URNAuthTokens() map[string]string { return m.urnAuthTokens }
-func (m *MockMsg) ContactName() string              { return m.contactName }
```

Comment:
don't need these because we're checking the values stored on the mock backend instead